templates/navigation-bar-new.twig

```diff
@@ -1,4 +1,9 @@
-<section id="header" class="top-navigation navbar {% if custom_styles.nav_border == 'border' %}navigation-bar_border{% endif %}">
+{% set header_classes = 'top-navigation navbar'
```

Comment:
@GP-Dan-Tovbein Can you remove this template instead? The only reason it was added is because it had the donate dropdown menu added. Since we're now adding such a menu ourselves, it's better if we remove this custom implementation. This is exactly what was already done in the [PR for NL](https://github.com/greenpeace/planet4-child-theme-netherlands/pull/47).

Reply:
You're right!